mixin/thanos/alerts/rule.libsonnet

```
@@ -153,6 +153,33 @@
               severity: 'warning',
             },
           },
+          {
+            alert: 'ThanosRuleNoEvaluationForLast10Intervals',
+            annotations: {
+              message: 'Thanos Rule {{$labels.job}} have {{ $value | humanize }}% rule groups that did not evaluate for at least 10x of their expected interval.',
+            },
+            expr: |||
+              time() - prometheus_rule_group_last_evaluation_timestamp_seconds{%(selector)s}
+                  >  10 * prometheus_rule_group_interval_seconds{%(selector)s}
```

Some spacing might be nice here to more quickly see what's happening.
```suggestion
              time() - prometheus_rule_group_last_evaluation_timestamp_seconds{%(selector)s}
              >
              10 * prometheus_rule_group_interval_seconds{%(selector)s}
```

---

we need PromQL fmt (: